WPFSKillTree/Utils/UrlProcessing/NaivePoeUrlDeserializer.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Text.RegularExpressions;
+using POESKillTree.Localization;
+
+namespace POESKillTree.Utils.UrlProcessing
+{
+    /// <summary>
+    /// Represents an object that extracts build information from urls.<para/>
+    /// This class uses nonstrict url check and consider specified url as compatible with the official planner https://pathofexile.com.
+    /// </summary>
+    public class NaivePoEUrlDeserializer : BuildUrlDeserializer
+    {
+        private readonly Regex _urlRegex = new Regex(@".*\/(?<build>[\w-=]+)");
+        private byte[] _rawData;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="NaivePoEUrlDeserializer"/> class.
+        /// </summary>
+        /// <param name="buildUrl">The PoE build url.</param>
+        public NaivePoEUrlDeserializer(string buildUrl) : base(buildUrl)
+        {
+        }
+
+        public override BuildUrlData GetBuildData()
+        {
+            var bytes = GetRawData();
+
+            var deserializedData = new BuildUrlData(BanditConverter.Default);
+            deserializedData.Version = bytes[0] << 24 | bytes[1] << 16 | bytes[2] << 8 | bytes[3];
+
+            if (!IsVersionSupported(deserializedData.Version))
+            {
+                throw new NotSupportedException(
```

Comment:
Again, we normally don't do this type of thing because people like to looks at builds between leagues as well as load older versions of builds.